### DISCUSSION TITLE
CFAST/Smokeview Vent Offset Visualization at Small Scales

### DISCUSSION THREAD

**Discussion by darynsagel** [firemodels/cfast#2138] [Q&A]
Hello, I am in a class where a group of us are simulating fire within a small compartment prop (around 1 meter on its longest side). When modeling this structure at its actual size in CFAST, the vent offset displays stretch far into the compartment spaces. When we model the same structure but scale its dimensions to be larger, using the same vent offset value makes the vents appear slimmer; it seems CFAST has a built-in lower limit for vent offset, and the spatial scale of the prop we are modeling accentuates the visual effects of this. Is there somewhere we can modify the vent offset lower limit for visualization purposes? Thank you!

**Comment by gforney** [comment]
upload your two test cases and I'll take a look

**Reply by darynsagel** [reply]
Images of the two test cases (actual size and 10x scaled version) are attached. Despite using relatively low vent offset values, there appears to be a hard-coded limit in either CFAST or Smokeview that causes them to appear much larger than the values we input. It is our understanding that this has no computational impact, but if we are able to modify it for visualization purposes then that would be fantastic!

It seems I cannot add the .in files we use in CFAST in GitHub, but I am happy to upload specific variable values, images of our input, or any other information as needed.

![Fire_box_Domain_scaled](https://github.com/user-attachments/assets/8b06ed86-274b-426b-bfa8-a51a8a13013e)
![Fire_box_Domain](https://github.com/user-attachments/assets/3efa8fd4-4e3a-4533-8b97-9c86730573c6)

**Comment by gforney** [comment]
Change the extension to .txt . You should be able to upload them then

**Reply by darynsagel** [reply]
Thank you! The .txt versions are attached below.

[Fire_box_scaled.txt](https://github.com/user-attachments/files/17780712/Fire_box_scaled.txt)
[Fire_box.txt](https://github.com/user-attachments/files/17780713/Fire_box.txt)

**Comment by gforney** [comment]
i'm not able to reproduce the problem.  I was using cfasg 7.7.5 and smokeview 6,9.4 .  try latest smokeview located at link below.  if it is still not working for you tell me what versions of cfast and smokeview your are using (for smokeview version type smokeview -v in a command shell)

https://github.com/firemodels/smv/releases/tag/SMV-6.9.4

**Reply by ritambhara1234** [reply]
We are using cfast 7.7.4 and smokeview 6.7.21 for this case.

**Comment by gforney** [comment]
use the latest smokeview at link above and report back if there is still a problem

**Comment by gforney** [comment]
this is the link for the latest smokeview, version 6.9.4
https://github.com/firemodels/smv/releases/tag/SMV-6.9.4

**Reply by ritambhara1234** [reply]
The two domains appear identical when viewed using Smokeview 6.9.4. Thank you, Dr. Forney!
